docs_src/design/adr/devops/0007-Release-Automation.md

```diff
@@ -12,14 +14,15 @@ EdgeX Foundry is a framework composed of microservices to ease development of Io
 
 ### Release Artifact Definition
 
-For the scope of Geneva release artifact types are defined as:
+For the scope of Hanoi release artifact types are defined as:
 
 - GitHub tags in the repositories.
 - Docker images in our Nexus repository and Docker hub.
-- Snaps in the Snapcraft store.
 
 This list is likely to expand in future releases.
 
+The building of snaps was removed from community scope in September 2020 but are still available on the [snapcraft store](https://snapcraft.io/edgexfoundry).
```

Comment:
Maybe?

The building and publishing of snaps was removed from community scope in September 2020 and is managed outside the community by Canonical.